docs/synapse/devopsguide.rst

```diff
@@ -292,6 +292,21 @@ log message pretty-printed log message::
       "user": "3189065f95d3ab0a6904e604260c0be2"
     }
 
+Custom date formatting strings can also be provided by setting the ``SYN_LOG_DATEFMT`` string. This is expected to be a
+strftime_ format string. The following shows an example of setting this value::
+
+    SYN_LOG_DATEFMT="%d%m%Y %H:%M:%S"
```

Comment:
Lets use `SYN_LOG_DATEFORMAT` here ( no need to rename internal vars or anything ). Clarity is worth more than brevity in this instance :)